### DISCUSSION TITLE
How to listen on specific address only?

### DISCUSSION THREAD

**Discussion by xlc** [microsoft/msquic#4610] [Q&A] [answered]
I cannot find example nor documents on how to listen on a specific address. There is `QuicAddrSetFamily` and `QuicAddrSetPort` but there is no such API to set the address.

Is there any examples or documents about this?

**Comment by nibanks** [comment, marked as answer]
Please see `PerfServer::Init` where we initialize the `LocalAddr` variable and then use it in `PerfServer::Start` in [PerfServer.cpp](https://github.com/microsoft/msquic/blob/main/src/perf/lib/PerfServer.cpp). We have a helper function `ConvertArgToAddress` from [msquichelper.h](https://github.com/microsoft/msquic/blob/main/src/inc/msquichelper.h).

**Reply by xlc** [reply]
Thanks. The helper functions are helpful. It will be better if they are more discoverable.
